doc/source/user/c-info.ufunc-tutorial.rst

```diff
@@ -644,7 +627,11 @@ np.float64(0.0)
 >>> npufunc.logit(a)
 array([       -inf, -1.09861229,  0.        ,  1.09861229,         inf])
 
+.. note::
 
+   Supporting ``float16`` (half-precision) in custom ufuncs is more complex
+   due to its non-standard C representation and conversion requirements.
+   For most users, starting with ``float32`` and ``float64`` is recommended.
```

Comment:
How about replacing this with something a bit more positive: "If you really need it, have a look at the numpy source code to see how it is done there."